src/foam/u2/borders/SearchBorder.js

```diff
@@ -0,0 +1,5 @@
+/**
+ * @license
+ * Copyright 2020 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */\n
```

Comment:
only header and no model definition?

Reply:
Yeah I had that in my index for over a month to remind me to work on it when I have time... removed it from this now